lib/ansible/modules/packaging/os/yum_repository.py

```diff
@@ -200,7 +200,7 @@
     default: 21600
   name:
     description:
-      - Unique repository ID.
+      - Unique repository ID. This option builds the section name of the repository in the .ini repo file.
```

Comment:
I think it's not necessary to mention the `.ini` here.

Reply:
@jtyr , I disagree, as I didn't know that yum repos were defined by .ini files, and I'm not familiar with that format. I believe that some users might use ansible because they are unfamiliar with the manual use of repo files, thus helping them understood its utilization. What I mean is: I don't think just writing "section name" will help new users finding where the value will appear. That being said, if you disagree with me, I'll remove it.